lib/improver/ensemble_calibration/ensemble_calibration.py

```diff
@@ -378,6 +384,71 @@ def __str__(self):
             self.distribution, self.desired_units,
             self.predictor_of_mean_flag, self.minimiser)
 
+    def create_coefficients_cube(
+            self, optimised_coeffs, current_forecast):
+        """Create a cube for storing the coefficients computed using EMOS.
+
+        .. See the documentation for examples of these cubes.
+        .. include:: extended_documentation/ensemble_calibration/
+           ensemble_calibration/create_coefficients_cube.rst
```

Comment:
Hooray! Thanks for using extended documentation. These examples make clear what is being produced when viewed in the documentation without cluttering the doc-string 👍

Reply:
🎈